src/main/resources/defaults/displayViews/persons/asideTemplate.html

```diff
@@ -64,7 +64,7 @@
       {{#if researcherOn}}
       <div class="mt-1 text-nowrap">
         <img src="assets/images/co_investigator_icon.png" height="20" width="20" alt="investigator network icon">
-        <span class="ml-1"><a href="{{vivoUrl}}/vis/investigator-network/{{id}}">Co-investigator Network</a></span>
+        <span class="ml-1"><a href="visualization/{{id}}/Co-investigator Network">Co-investigator Network</a></span>
```

Comment:
These URLs seem wrong to me.

Should there really be "Co-author`SPACE`Network"?
Are we also using case-sensitive names?

This just doesn't seem normaly to me.
If this is the expected URL, then fine.

Reply:
It is expected. It works as is but may be better to be explicit and use `+` or `%20`.

Reply:
Either is fine with me. `+` is easier to read.

Reply:
Done.